docs/getting-started/getting-started-with-services.rst

```diff
@@ -86,7 +85,3 @@ Once you have a version, you can now publish your service! Publishing will creat
 .. figure:: /assets/images/docs/services-public-page.png
 
 Now other users can view and :doc:`star <../end-user-topics/starring>` your service. You will also have the option to add published services to a :doc:`collection <../advanced-topics/organizations-and-collections>`.
-
-
-.. discourse::
-    :topic_identifier: 1970
```

Comment:
Did you mean to remove the discourse embed?

Reply:
totally did not mean to. Added it back.